HACKING.md

```diff
@@ -39,3 +39,42 @@ To run the unit tests:
 To run the integration tests:
 
     ./integration-test.sh
+
+## Generating Stats
+
+To generate the stats on your local machine you will need to import the listens into HDFS. 
+You should use the ``listenbrainz-incremental-dumps`` for your local environment as the size of 
+``listenbrainz-full-export-dumps`` is large for a local machine (The latest 
+``listenbrainz-full-export-dump`` size is around 9.3 GB). Follow the given steps to import the 
+listens.
+
+### Update config.py
+
+Open the ``config.py`` file present in the ``listenbrainz_spark`` directory and update the FTP 
+listens directory to import the incremental dumps as given.
+
+    FTP_LISTENS_DIR = '/pub/musicbrainz/listenbrainz/incremental/'
+
+Now update the temporary listens directory and dump with the directory name and the .tar.xz file 
```

Comment:
update TEMP_LISTENS_DIR with the directory name and TEMP_LISTENS_DUMP with the .tar.xz file?

Also, where would they find the directory and tar name?

Reply:
I will add the ftp server link